src/pt/lermangaonline/src/eu/kanade/tachiyomi/extension/pt/lermangaonline/LerMangaOnline.kt

```diff
@@ -0,0 +1,120 @@
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.concurrent.TimeUnit
+
+class LerMangaOnline : ParsedHttpSource() {
+    override val name = "Ler Mangá Online"
+
+    override val baseUrl = "https://lermangaonline.com.br"
+
+    override val lang = "pt-BR"
+
+    override val supportsLatest = true
+
+    override val client: OkHttpClient = network.cloudflareClient.newBuilder()
+        .rateLimitHost(baseUrl.toHttpUrl(), 1, 1, TimeUnit.SECONDS)
+        .build()
+
+    override fun chapterFromElement(element: Element) = SChapter.create().apply {
+        name = element.selectFirst("div.capitulo")!!.ownText()
+        date_upload = (element.selectFirst("span")?.text() ?: "").toDate()
+        setUrlWithoutDomain(element.absUrl("href"))
+    }
+
+    override fun chapterListSelector() = "div.capitulos a"
+
+    override fun imageUrlParse(document: Document) = ""
+
+    override fun latestUpdatesFromElement(element: Element) = SManga.create().apply {
+        title = element.selectFirst("section h3")!!.text()
+        thumbnail_url = element.selectFirst("div.poster img")?.absUrl("src")
+        setUrlWithoutDomain(element.selectFirst("div.poster a")!!.absUrl("href"))
+    }
+
+    override fun latestUpdatesNextPageSelector() = "div.wp-pagenavi [aria-current] + a"
+
+    override fun latestUpdatesRequest(page: Int): Request {
+        val url = "$baseUrl/capitulo/page/$page".toHttpUrl().newBuilder()
+            .build()
+        return GET(url, headers)
+    }
+
+    override fun latestUpdatesSelector() = "div.box-indx section.materias article"
+
+    override fun mangaDetailsParse(document: Document) = SManga.create().apply {
+        val info = document.selectFirst("div.box-flex")
+        title = info!!.selectFirst("div.sinopse a")!!.text()
+        description = info.selectFirst("div.sinopse div:nth-child(2)")?.text()
+        thumbnail_url = info.selectFirst("div.poster img")?.srcAttr()
+        genre = document.select("div.categorias-blog a").joinToString { it.text() }
+        status = SManga.UNKNOWN
+    }
+
+    override fun pageListParse(document: Document): List<Page> {
+        val elements = document.select("div.images img")
+        return elements.mapIndexed { i, el ->
+            Page(i, imageUrl = el.srcAttr())
+        }
+    }
+
+    override fun popularMangaFromElement(element: Element) = latestUpdatesFromElement(element)
+
+    override fun popularMangaNextPageSelector() = latestUpdatesNextPageSelector()
+
+    override fun popularMangaRequest(page: Int): Request {
+        val url = "$baseUrl/page/$page".toHttpUrl().newBuilder()
+            .build()
+        return GET(url, headers)
+    }
+
+    override fun popularMangaSelector() = latestUpdatesSelector()
+
+    override fun searchMangaFromElement(element: Element) = latestUpdatesFromElement(element)
+
+    override fun searchMangaNextPageSelector() = latestUpdatesNextPageSelector()
```

Comment:
If you're going to have Popular and Latest show the same thing, it's better to implement Popular only and make `supportsLatest = false`.

Reply:
The element selectors are the same

Reply:
> The element selectors are the same

My bad, I didn't notice the URLs were different.